src/shared/bootspec.c

```diff
@@ -1145,6 +1145,8 @@ int boot_config_augment_from_loader(
                 "auto-windows",                  "Windows Boot Manager",
                 "auto-efi-shell",                "EFI Shell",
                 "auto-efi-default",              "EFI Default Loader",
+                "auto-poweroff",                 "Power Off The System",
+                "auto-reboot",                   "Reboot The System",
                 "auto-reboot-to-firmware-setup", "Reboot Into Firmware Interface",
```

Comment:
I'm not sure about the capitalization of "The". But we also capitalize "Into", and those should be the same.